src/@chakra-ui/gatsby-plugin/foundations/typography.ts

```diff
@@ -0,0 +1,19 @@
+const typography = {
+  // TODO: Uncomment when the `Inter` font is included in the project
+  // fonts: {
+  //   heading: "Inter, sans-serif",
+  //   body: "Inter, sans-serif",
+  // },
```

Comment:
Cool. Curious, is there any difference between defining it here than on the top-level (as it were before)?

Anyway, I like more placing this here than in the old place :)

Reply:
No difference, just for code splitting! 😄

Reply:
Cool haha didn't notice that you were spreading the props on the other side 👍🏼 this is better